tools/wasi/wasi.cpp

```diff
@@ -1,7 +1,8 @@
 // Fizzy: A fast WebAssembly interpreter
-// Copyright 2019-2020 The Fizzy Authors.
+// Copyright 2020 The Fizzy Authors.
 // SPDX-License-Identifier: Apache-2.0
 
+#include "wasi.hpp"
 #include "execute.hpp"
 #include "limits.hpp"
 #include "parser.hpp"
```

Comment:
Is the `<iostream>` header still needed here (since `std::cerr` is moved to `main.cpp`)?